src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+const toNumber = (string) => {
+  if (typeof string === 'string') {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum);
+  }
+};
+
+tableHeader.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeader.children];
+  const columnName = e.target.closest('th');
```

Comment:
The 'closest' method will return null if no matching element is found. You should check if 'columnName' is not null before proceeding to use it.